partcad/src/partcad/runtime_python.py

```
@@ -16,10 +16,47 @@
 
 from . import runtime
 from . import logging as pc_logging
+from . import sync_threads
+
+
+class VenvLock(object):
```

Ruff (0.8.2)

Class `VenvLock` inherits from `object`

Remove `object` inheritance

(UP004)